tests/backend/runner/test_base.py

```diff
@@ -0,0 +1,106 @@
+import re
+from typing import Any
+
+import numpy as np
+import pytest
+
+from letstune.backend.runner._base import normalize_metric_values
+
+
+def is_normalized(metric_values: Any) -> bool:
+    if type(metric_values) is not dict:
+        return False
+
+    for k, v in metric_values.items():
```

Comment:
I suggest refactoring with `all(... for k, v in metric_values.items())` function.